### ISSUE TITLE
[Minor bug] Key errors when using functionality of label layer

### ISSUE THREAD

**Issue by melonora** [napari/napari#5535]
## 🐛 Bug

In Napari when having created a label layer and upon switching to either erase, paint, fill, picker or zoom, a key error is given. 
This is due to the key not being present in the _actions dictionary of the ActionManager of napari.utils. One example of the error is
![afbeelding](https://user-images.githubusercontent.com/22346363/217188493-430b0d82-8794-4ef4-a0af-da2b2f1468f4.png)
 
In this case the key should have been napari:activate_labels_paint_mode (see image below). Also for the other cases adding labels in the keyword would fix this issue. The functionality does not seem to have changed. Transform, erase, .... seem to still work, thus it seems to be a minor bug. It seems to be introduced with the recent merge of #4894.
![afbeelding](https://user-images.githubusercontent.com/22346363/217191384-fbbd7213-bcf0-4f42-9b57-8036b9a2ec56.png)

## To Reproduce

1. Start Napari
2. Create labels layer
3. Switch to either one of the modes described above.

## Expected behavior

No key errors.

## Environment

napari: 0.4.17rc4.dev180+gd7b22b19.d20230125
Platform: Windows-10-10.0.22621-SP0
Python: 3.9.13 | packaged by conda-forge | (main, May 27 2022, 16:51:29) [MSC v.1929 64 bit (AMD64)]
Qt: 5.15.2
PyQt5: 5.15.7
NumPy: 1.23.4
SciPy: 1.9.3
Dask: 2022.11.0
VisPy: 0.12.1
magicgui: 0.6.0
superqt: 0.4.0
in-n-out: 0.1.6
app-model: 0.1.1
npe2: 0.6.1

OpenGL:
- GL version: 4.6.0 NVIDIA 526.98
- MAX_TEXTURE_SIZE: 32768

Screens:
- screen 1: resolution 1920x1080, scale 1.0

Settings path:
- C:\Users\w-mv\AppData\Local\napari\napari_e081f750b90253ee1b4c7328e2bd7ed6bfeabc20\settings.yaml

**Comment by melonora** [comment]
in ```qt_labels_controls.py``` inserting labels fixes the issue partially
```
action_manager.bind_button(
            'napari:activate_fill_mode',
            self.fill_button,
        )
```

**Comment by AhmetCanSolak** [comment]
This issue seems to be addressed in #5539, shall we close it @Czaki?
